File: Problem_set1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void){
    float n  ;
    int count = 0 ;
// get the change from user
    do
        {
            n = get_float("Change owed: ");
        } 
    while (n < 0);
// convert to cents
int coins = round(n*100);
// count the less number of change
while (coins != 0){
    if (coins >= 25 ){
        count++;
        coins-=25;
    }
    else if (coins >= 10 ){
        count++;
        coins-=10;
    }
    else if (coins >= 5 ){
        count++;
        coins-=5;
    }
    else if (coins >= 1 ){
        count++;
        coins-=1;
    }
}
printf("%i\n",count);
}

File: Problem_set6/cash.py
from cs50 import *
from math import *

# make sure that the change not negative just positive number
while True:
    n = get_float("Change owed: ")
    if (n >= 0):
        break
    
count = 0 
coins = round(n * 100)

# calculate cash that will be paid
while (coins != 0):
    if (coins >= 25 ):
        count += 1
        coins -= 25
        
    elif (coins >= 10 ):
        count += 1
        coins -= 10
    
    elif (coins >= 5 ):
        count += 1
        coins -= 5

    elif (coins >= 1 ):
        count += 1
        coins -= 1 
    
print(count)
